lib/Twig/Extension/AssetHelperExtensions.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Twig\Extension;
+
+use Pimcore\Model\Asset;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigTest;
+
+final class AssetHelperExtensions extends AbstractExtension
+{
+    public function getTests(): array
+    {
+        return [
+            new TwigTest('pimcore_asset', static function ($object) {
+                return is_object($object) && $object instanceof Asset;
```

Comment:
The `is_object` check should not be needed. 
The PHP docs say: 
`instanceof does not throw any error if the variable being tested is not an object, it simply returns FALSE. Constants, however, were not allowed prior to PHP 7.3.0.`

```suggestion
                return $object instanceof Asset;
```

Reply:
true, not sure why I used that here, it for sure had a reason :) but might be to older PHP Versions, will remove it.